File: iotgateway/src/main/AngularClient/src/app/Entities/Charger.ts
export class Charger {
  public ChargePowerPercentage: number;
  public ChargeCurrent: number;
  public ChargeVoltage: number;
  public PreviousState: boolean;
  public ChargerIsOn: boolean;
  public OldChargerPowerPercentage: number;
  public BatteryLimitPercentage: number;
  public SetBatteryLimitPercentage: boolean;
}

File: charger/src/main/java/com/mobilecomputing/actuator/Entities/Charger.java
package com.mobilecomputing.actuator.Entities;

public class Charger {
    public int ChargePowerPercentage;
    public float ChargeCurrent;
    public float ChargeVoltage;
    public boolean PreviousState;
    public boolean ChargerIsOn;
    public int OldChargerPowerPercentage;
    public int BatteryLimitPercentage;
    public boolean SetBatteryLimitPercentage;

    public Charger() {
        ChargerIsOn = false;
        ChargePowerPercentage = 100;
        ChargeCurrent = 30;
        ChargeVoltage = 230;
        OldChargerPowerPercentage = ChargePowerPercentage;
        PreviousState = ChargerIsOn;
        BatteryLimitPercentage = 100;
        SetBatteryLimitPercentage = false;
    }
}
